package/cfg/config.yaml

```diff
@@ -191,14 +191,11 @@ managedservices:
 version_mapping:
   "rke-1.13": "rke-cis-1.4"
   "rke-1.14": "rke-cis-1.4"
-  "rke-1.15": "rke-cis-1.5-hardened"
-  "rke-1.16": "rke-cis-1.5-hardened"
-  "rke-1.17": "rke-cis-1.5-hardened"
-  "rke-1.18": "rke-cis-1.5-hardened"
   "rke-1.15": "rke-cis-1.5-permissive"
-  "rke-1.16": "rke-cis-1.5-permissive"
-  "rke-1.17": "rke-cis-1.5-permissive"
-  "rke-1.18": "rke-cis-1.5-permissive"
+  "rke-1.16": "rke-cis-1.6-permissive"
+  "rke-1.17": "rke-cis-1.6-permissive"
+  "rke-1.18": "rke-cis-1.6-permissive"
+  "rke-1.19": "rke-cis-1.6-permissive"
   "v1.18.10+rke2r1": "rke2-cis-1.5-hardened"
```

Comment:
@briandowns While reviewing this realised that we need to set only one profile as the default for each k8s version. Thus this should change for rke2 to not have both permissive and hardened mapped to "v1.18.10+rke2r1"

Reply:
Thank you for the heads up. I'll update this and test in my next PR unless you're prefer it in a separate PR that addresses only this.